jOOQ-examples/jOOQ-spring-boot-example/pom.xml

```diff
@@ -11,7 +11,7 @@
 
     <groupId>org.jooq</groupId>
     <artifactId>jooq-spring-boot-example</artifactId>
-    <version>${org.jooq.version}</version>
+    <version>${jooq.version}</version>
```

Comment:
I think it's better using the "spring" property because that's mimics the stuff you get from start.spring.io which is the recommend way with the Spring Boot BOM. You could set jooq.version to the value of org.jooq.version… But i'd remove the explicit version in the dependencies… IMHO the usage of the Spring Boot BOM is highly recommend to get version of stuff that works well together.